# msg/TaskPriority.msg
int32[] task_ids
float64[] priorities
bool[] on_critical_path
float64[] slack_times